.github/CONTRIBUTING.md

```diff
@@ -37,17 +37,18 @@ Please consider the following :
 	- Add font name to `UPPERCASE_FONTS`
 6. If font is Non-ASCII:
 	- Add font name to `TEST_FILTERED_FONTS` list
-	- Add font name to `RANDOM_FILTERED_FONTS` list
 	- Add a test case to `test2.py` (*Alphabetical order*)
 7. If font is ASCII:
 	- Add a test case to `test.py` (*Alphabetical order*) 
-8. Re-run `FontList.ipynb`
+8. If you want to remove this font from shuffle list: (*Optional*)
+	- Add font name to `RANDOM_FILTERED_FONTS` list 
```

Comment:
I can't find it's logic :confused:
if we want to remove font from shuffle list we should add font to `RANDOM_FILTERED_FONTS`?
or if we want to add font to shuffle list we should do this?